src/hub/atlas.js

```
@@ -421,6 +421,73 @@ class AtlasClient {
         return users;
     }
 
+    /**
+     * Update an existing users data
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint PATCH method
+     * 
+     * @returns {Boolean} indicates whether the patch was successful or not
+     */
+    async patchUser(data) {
+        const op = { method: "PATCH", body: { ...data } };
+        try{
+            await this.fetch("/v1/user/" + data.id, op);
+            return true;
+        }catch (err){
+            console.log(err);
+            return false;
+        }
+    }
+
+    /**
+     * Add a new user to the existing user set
+     *
+     * @param {Object} data - A user data object as specified for the v1/user/ endpoint POST method
+     * 
+     * @returns {Boolean} indicates whether the post was successful or not
+     */
+    async postUser(data) {
+        const op = { method: "POST", body: { ...data } };
+        try{
+            await this.fetch("/v1/user/", op);
+            return true;
+        }catch (err){
+            console.log(err);
+            return false;
+        }
+    }
+
+    /**
+     * Register a new user with atlas
+     *
+     * @param {Object} body - A user data object in the form:
+     *  {   
+     *      captcha, 
+     *      phone, 
+     *      email, 
+     *      fullname, 
+     *      tag_slug, 
+     *      password, 
+     *      org_name, 
+     *      org_slug 
+     * }
+     * All fields are required except phone
+     * 
+     * Use the site key 6Lcr4JMUAAAAAOljN5puqdFrcVeCyexMNHlWtWHX as the seed
+     * for your recaptcha token.
+     * 
+     * @returns {Object} - {nametag, orgslug, jwt}
+     */
+    async postJoin(body) {
+        try{
+            const result = await this.fetch("/v1/join/", { method: "POST", body });
+            return result;
+        }catch (err){
+            console.log(err);
```

Same as above.  Fail harder.